test/solidity/Facets/AccessManagerFacet.t.sol

```diff
@@ -67,4 +71,87 @@ contract AccessManagerFacetTest is TestBase {
         vm.prank(address(0xb33f));
         restricted.restrictedMethod();
     }
+
+    function testRevertCannotAuthorizeSelf() public {
+        vm.expectRevert(CannotAuthoriseSelf.selector);
+        accessMgr.setCanExecute(
```

Comment:
please add blank lines between expectRevert and the actual call.
Try to find a consistent approach, pls.

Reply:
We should keep it somehow consistent...I admit that it's currently not but maybe we can work out some rules (and often-used examples) together that make most sense and then adhere to it.